gradle.properties

```diff
@@ -18,22 +18,22 @@ kotlinJvmVersion = 1.8
 kotlinVersion = 1.3.72
 kotlinCoroutinesVersion = 1.3.8
 
-classGraphVersion = 4.8.87
+classGraphVersion = 4.8.90
 graphQLJavaVersion = 15.0
 jacksonVersion = 2.11.3
 kotlinPoetVersion = 1.6.0
 ktorVersion = 1.3.1
-reactorVersion = 3.3.10.RELEASE
-reactorExtensionsVersion = 1.0.2.RELEASE
-springBootVersion = 2.3.4.RELEASE
-springVersion = 5.2.9.RELEASE
+reactorVersion = 3.4.0
+reactorExtensionsVersion = 1.1.0
+springBootVersion = 2.3.5.RELEASE
+springVersion = 5.2.10.RELEASE
 
 # test dependency versions
-junitVersion = 5.6.2
-mockkVersion = 1.10.0
-mustacheVersion = 0.9.6
-rxjavaVersion = 3.0.4
-wireMockVersion = 2.26.3
+junitVersion = 5.7.0
```

Comment:
I believe we were hitting some issues in the past with `5.7.0` as Spring test is dependent on `5.6.3`